strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/main/java/org/carlspring/strongbox/providers/layout/NpmUnpublishService.java

```diff
@@ -0,0 +1,146 @@
+package org.carlspring.strongbox.providers.layout;
+
+import org.carlspring.strongbox.artifact.coordinates.NpmArtifactCoordinates;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.carlspring.strongbox.repository.NpmRepositoryFeatures;
+import org.carlspring.strongbox.services.ArtifactManagementService;
+import org.carlspring.strongbox.services.ArtifactResolutionService;
+import org.carlspring.strongbox.storage.repository.Repository;
+
+import javax.inject.Inject;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author mpoznyak
+ */
+@Component
+public class NpmUnpublishService
+{
+
+    public enum Result
+    {ARTIFACT_NOT_EXISTS, INTERNAL_SERVER_ERROR, UNPUBLISHED, UNPUBLISH_DISABLED}
+
```

Comment:
Could you please change this to not be a one-liner?
```jave
     public enum Result
    {ARTIFACT_NOT_EXISTS, INTERNAL_SERVER_ERROR, UNPUBLISHED, UNPUBLISH_DISABLED}
```
Like this:
```
    public enum Result
    {
        ARTIFACT_NOT_EXISTS,
        INTERNAL_SERVER_ERROR,
        UNPUBLISHED,
        UNPUBLISH_DISABLED
    }
```
And could you please rename `ARTIFACT_NOT_EXISTS` to `ARTIFACT_DOES_NOT_EXIST`?

Thanks! :)